timemachine/fe/dummy.py

```diff
@@ -0,0 +1,618 @@
+from email.headerregistry import ContentDispositionHeader
+from operator import length_hint
+from re import L
+import numpy as np
+import networkx as nx
+import copy
+import itertools
+
+
+def _add_successors(mol, core, groups):
+    # (ytz): internal utility used for bfs
+    next_groups = []
+    for group in groups:
+        # grab the last node
+        last_node = group[-1]
+        for nbr in mol.GetAtomWithIdx(last_node).GetNeighbors():
+            new_group = copy.deepcopy(group)
+            nbr = nbr.GetIdx()
+            # ensure is core_atom and not already visited
+            if nbr in core and nbr not in new_group:
+                new_group.append(nbr)
+                next_groups.append(new_group)
+
+    return next_groups
+
+
+def identify_anchor_groups(mol, core, anchor):
+    """
+    Generate putative choices for valid anchor groups. An anchor group
```

Comment:
Do you mean that the choices are putatively valid? Wasn't familiar with the word, but `putative choices` doesn't quite make sense to me